packages/react-components/src/molecules/__tests__/Discussion.test.tsx

```diff
@@ -60,3 +62,77 @@ it('removes reply modal when user clicks cancel button', () => {
 
   expect(queryByText(/Reply to quick check/i)).not.toBeInTheDocument();
 });
+
+describe('when there are replies', () => {
+  describe('when collapsed', () => {
```

Comment:
We don't need this to be wrapped in this `when collapsed` describe, right?

Reply:
Yea, agreed. Removing it